main.go

```diff
@@ -68,15 +69,15 @@ service:
   pipelines:
     traces:
       receivers: [otlp]
-      processors: []
+      processors: [lint]
```

Comment:
maybe make a flag for this as when you are developing a new semver, it may be clutter to see incompatabilities. I don't mind if it is on by default.

Reply:
I would love to one day have a choice like semver=1.27.0 (vs one before that) because it takes a month or two for some changes to break free. We don't need to take action here now though. Just a thought